wallbox_client.py

```diff
@@ -94,7 +128,10 @@ def set_charger_to_autostart_on_connect(self, setting: str):
         time.sleep(self.args["wait_between_charger_write_actions"] / 1000)
 
     def set_charger_action(self, action: str):
-        """Set action to start/stop charging."""
+        # FNC: hopefully not needed
+        # global last_restart
```

Comment:
Let's remove these comments about not needing global variables. We have seen the code run successfully in production, with the globals commented out.